watertap/data/techno_economic/suboxic_activated_sludge_process.yaml

```diff
@@ -0,0 +1,89 @@
+default:
+  energy_electric_flow_vol_inlet:
+    value: 0.5
+    units: kWh/m^3
+  recovery_frac_mass_H2O:
+    value: 1
+    units: dimensionless
+  default_removal_frac_mass_solute:
+    value: 0
+    units: dimensionless
+  removal_frac_mass_solute:
+    bod:
+      value: 0.9682
+      units: dimensionless
+      constituent_longform: Biological Oxygen Demand (BOD)
+    tss:
+      value: 0.9851
+      units: dimensionless
+      constituent_longform: Total Suspended Solids (TSS)
+    tkn:
+      value: 0.8846
+      units: dimensionless
+      constituent_longform: Total Kjeldahl Nitrogen (TKN)
```

Comment:
I know that you're following info provided, but when we go to make flowsheets and think about the situation where we mix and match different unit models, if all other models include something like `nitrogen` or `ammonia_as_nitrogen`, having this component as TKN for this unit might not be desirable.

Reply:
The TKN is defined as the sum of organic nitrogen and ammonia, while the total nitrogen include TKN plus nitrate nitrogen and nitrite nitrogen. So `tkn` is a component of `nitrogen`, and `ammonia_as_nitrogen` is a component of `tkn`.

Reply:
@adam-a-a's point is more general than that - there are many ways that we can go about defining nitrogen in a waste stream, and if we start using specific ones in every model then there is no way to link them all together. Thus, we have tended to take a more hands-off approach and just say "nitrogen" or "ammonia" and leave the finer details of how that is measured up to the user (who needs to set the concentrations and removal fractions accordingly).

I.e. if one unit uses `tdk_nitrogen` and another uses `my_nitrogen`, then the user will have to do some sort of manipulation to get the two to work together. If they both use `nitrogen`, then the user only needs to set the parameters according (which they should be doing anyway).

Reply:
@andrewlee94 @adam-a-a Thank you for clarification, I understand the points right now. This amo team only provides the removal fraction for `tkn` at present. If the user wants to make a comparison, they should provide extra information for calculating `nitrogen` or `ammonia_as_nitrogen` accordingly based on their own data.